wiki/fr/fr-Getting-Started.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: wiki
+title: "Pour commencer"
+lang: "fr"
+permalink: "/wiki/Getting-Started"
+---
+
+# Commencer à utiliser Jamulus
+
+## Vous avez besoin de
+
+1. **Un ordinateur** au moins sous **Windows 10**, **MacOS X** El Capitan (v10.11) ou plus réçent, ou **GNU/Linux**.
+1. **Une connection Internet filaire** (n'utilisez pas la Wi-Fi, cela va causer des problèmes de son). La plupart des connexions standard suffisent[^1].
+1. **Des microphones filaires - pas de haut parleurs** pour éviter de l'écho ou des larsen et pour minimiser le délai[^2].
+
+## Maximiser la qualité et diminuer le délai
+
+### 1. Utilisez une interface audio∕un microphone externe, pas votre carte son interne
+
+Sauf dans le cas où vous êtes un musicien ayant un microphone (USB), il est fortement recommandé d'utiliser une carte son : un appareil sur lequel vous pouvez brancher votre instrument. Celle-çi se connectera à votre PC, généralement par USB. Visitez [la section matériel](Hardware-Setup).
+
+### 2. Réduisez votre trafic Internet et fermez les programmes de votre PC
+
+Assurez-vous que Jamulus ait le maximum de ressources sur votre machine ainsi que via votre connexion réseau quand vous l'utilisez.
+
+### 3. Connectez-vous aux serveurs qui ont le meilleur temps de réponse
+
+Les serveurs qui s'affichent dans la liste avec un temps de 15ms ou moins optimiserons votre expérience (même s'il y a d'autres éléments qui influent sur les performances). Avec des temps de réponse plus longs, c'est à vos risques et périls.
+
+Maintenant que vous avez les bases, c'est parti :
+
+<div class="fx-row fx-row-start-xs button-container">
+  <a href="Installation-for-Windows" class="button fx-col-100-xs">Installer sur Windows</a>
+  <a href="Installation-for-Macintosh" class="button fx-col-100-xs">Installer sur macOS</a>
+  <a href="Installation-for-Linux" class="button fx-col-100-xs">Installer sur une distribution GNU/Linux</a>
+</div>
+
+Découvrez aussi : [Jamulus OS !](https://sourceforge.net/projects/jamulus-os/files/JamulusOS/) Exécutez Jamulus sur n'importe quel ordinateur avec une clé USB (uniquement pour les systèmes en 64 bits). Vérifiez la compatibilité de votre matériel et profitez-en !
+
+## Tout est bon ?
+
+Si vous voulez configurer Jamulus, visitez [la section matériel](Hardware-Setup).
+
+Pour de plus amples informations à propos de l'utilisation de Jamulus, regardez [le mode d'emploi](https://github.com/corrados/jamulus/blob/master/src/res/homepage/manual.md).
+
+## D'autres guides
+* [Cet excellent guide (en)](https://www.facebook.com/notes/jamulus-online-musicianssingers-jamming/idiots-guide-to-jamulus-app/510044532903831/){: target="_blank" rel="noopener noreferrer"} par [Simon Tomlinson](https://www.facebook.com/simon.james.tomlinson?eid=ARBQoY3KcZAtS3pGdLJuqvQTeRSOo4gHdQZT7nNzOt1oPMGgZ4_3GERe-rOyH5PxsSHVYYXjWwcqd71a){: target="_blank" rel="noopener noreferrer"} (_Facebook_) pourrait aussi vous aider.
+
+## Un ou des problème(s) ? Ça ne fonctionne pas longtemps ?
```

Comment:
Accents and punctuation can cause problems for internal links.

Reply:
> I don't think so. Any example? If ever it would be the case, then we would need to fix it, not removing accent in French (and other non-english language).

When I worked on the translation (the pages in #41) I had some internal links wich wasen't working.
Not sure if it was accents or puctuation.
I'll make some other tests…

Reply:
If that is the case, then we (all the languages) have a problem and we would need to discuss it in a wider thread than here. Think about potential Cyrillic, Arabic, Indian, Japanese, Chinese...etc translations.

Let's keep that outside of this thread and open an new issue if that is the case.

Reply:
> If that is the case, then we (all the languages) have a problem and we would need to discuss it in a wider thread than here. Think about potential Cyrillic, Arabic, Indian, Japanese, Chinese...etc translations.
> 
> Let's keep that outside of this thread and open an new issue if that is the case.

For the french, I've found what was wrong. Punctuation is ignored, but not spaces. (Can't test for the other languages)

For the title

```
## Vous avez des problèmes ? vous ne parvenez pas à jouer en rythme ?
```

The link must be
```
[ces notes](Getting-Started#vous-avez-des-problèmes--vous-ne-parvenez-pas-à-jouer-en-rythme-)
```

One « - » by spaces, don't forgot the space before the last « ? ».

Reply:
OK, thank you very much for doing this research.